cli.js

```diff
@@ -28,9 +29,10 @@ program.command('translate')
       for await (const language of languages) {
         console.time(`Generate translation ${text} ${language.name}`);
         const { text: result } = await translate(text, { from: sourceLanguage, to: language.name });
+
         language.files.forEach(fileName => {
-          const { file: languageJson, parsedPath } = getOrCreateTranslationFile(basePath, fileName);
-          languageJson[nameOfTranslation] = result;
+          let { file: languageJson, parsedPath } = getOrCreateTranslationFile(basePath, fileName);          
```

Comment:
NIT:
Extra space
```suggestion
          let { file: languageJson, parsedPath } = getOrCreateTranslationFile(basePath, fileName);
```

Reply:
Why changed to const?

Reply:
Nice catch! I did the change before of using dset, then I miss change it again for const. Will change it